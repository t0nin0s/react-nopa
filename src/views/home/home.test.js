import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Home from './home'

test('Home component renders as expected', () => {
  const component = shallow(<Home />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
