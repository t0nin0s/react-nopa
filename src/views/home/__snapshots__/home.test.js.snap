// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders as expected 1`] = `
<Layout
  home={true}
>
  <div
    className="content"
  >
    <section
      className="section section--primary"
    >
      <div
        className="container centered"
      >
        <h1
          className="section__h1 section__h1--margin"
        >
          Your finances, in one place
        </h1>
        <p
          className="section__p"
        >
          Track all your payments by connecting as
                      many bank accounts as you'd like to your
                      NOPA account and get updates your balance
                      instantly.
        </p>
        <Link
          className="button button--large"
          replace={false}
          to="/bank"
        >
          Get Started
        </Link>
      </div>
    </section>
    <section
      className="section section--secondary"
    >
      <div
        className="two-columns"
      >
        <div
          className="column--desktop centered"
        >
          <h1
            className="columns__h1"
          >
            There's no such things as 'one size fits all' finance.
          </h1>
          <p
            className="columns__p"
          >
            We were founded to make money simple and fair,
                            for everyone: whether you're looking for a loan,
                            or to get better rewards for your investment.
          </p>
        </div>
        <div
          className="column--desktop centered"
        >
          <img
            alt=""
            src={Object {}}
          />
        </div>
      </div>
    </section>
    <Partners />
  </div>
</Layout>
`;
